src/transaction/log_record.hpp

```diff
@@ -230,11 +230,16 @@ struct log_rec_replication
   int rcvindex;
 };
 
+typedef INT64 time_msec_t;
+
 /* Log the time of termination of transaction */
 typedef struct log_rec_donetime LOG_REC_DONETIME;
 struct log_rec_donetime
 {
-  INT64 at_time;		/* Database creation time. For safety reasons */
+  time_msec_t at_time;		/* Database creation time. For safety reasons .. LE: what?
+                                 * NOTE: stored as milliseconds with the value as accurate
+                                 * as possible as provided via system structs such as
+                                 * 'timeval' or 'timespec' */
```

Comment:
The original comment seems outdated, replace it. Please keep comments brief.
```suggestion
  time_msec_t at_time;		/* Transaction commit time */
```

Reply:
Done